File: mycollab-web/src/main/scala/com/mycollab/module/user/accountsettings/view/events/AccountBillingEvent.scala
package com.mycollab.module.user.accountsettings.view.events

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd
  * @since 5.0.5
  */
object AccountBillingEvent {

  class CancelAccount(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoSummary(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoHistory(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}

File: mycollab-web/src/main/java/com/mycollab/module/user/accountsettings/view/event/AccountBillingEvent.kt
package com.mycollab.module.user.accountsettings.view.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class AccountBillingEvent {
    class CancelAccount(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoSummary(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoHistory(source: Any, data: Any?) : ApplicationEvent(source, data)
}